src/main/java/com/airlines/database/JDBCConnection.java

```diff
@@ -0,0 +1,27 @@
+package com.airlines.database;
+
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.SQLException;
+import java.util.ResourceBundle;
+
+public class JDBCConnection {
+
+    private Connection getDBConnection() {
+
+        ResourceBundle resourceBundle = ResourceBundle.getBundle("db");
+        String url = resourceBundle.getString("url");
+        String username = resourceBundle.getString("username");
+        String password = resourceBundle.getString("password");
+
+        Connection dbConnection = null;
+        try {
+            dbConnection = DriverManager.getConnection(url, username, password);
```

Comment:
For better performance I recommend using Connection pool approach, no need for custom implementation, you could use HikariDatasource, so first, you should init this DataSource and assign it to private field, and then method getConnection would invoke this dataSource field .getConnection.

Reply:
done in https://github.com/pielena/airlines-jdbc-app/commit/adc21a684639bfe2ff23f0909b6108fa56f949e9